tests/conftest.py

```diff
@@ -283,73 +283,74 @@ def client(
     _raw_client.reset_debug_features()
     _raw_client.open()
     try:
```

Comment:
Maybe we should use `contextlib.closing(_raw_client)` instead?

Reply:
that would make more sense if we _already_ had a context manager, which would wrap this
but if we did we'd just close in that context manager
so i think try/finally is more readable here